velox/type/Tokenizer.cpp

```diff
@@ -54,17 +57,17 @@ std::unique_ptr<Subfield::PathElement> Tokenizer::computeNext() {
     return nullptr;
   }
 
-  if (tryMatch(DOT)) {
+  if (tryMatchSeparator(DOT)) {
```

Comment:
Can we use `separators_.dot` here?  We can remove the constant `DOT`, `QUOTE`, etc. from this class now I think

Reply:
Thanks for the suggestion. Updated.